src/leiningen/new/onyx_app/README.md

```diff
@@ -172,7 +173,7 @@ You should now see results streaming into MySQL
 {{/docker?}}
 ## License
 
-Copyright © 2015 FIXME
+Copyright © 2015 Distributed Masonry 
```

Comment:
I think this should actually remain as `FIXME`, as it's the templated version of the user's README.